src/tls/Makefile-tls.am

```diff
@@ -0,0 +1,64 @@
+libexec_PROGRAMS += cockpit-tls
+
+cockpit_tls_SOURCES =					\
+	src/common/cockpitmemory.c			\
```

Comment:
I feel like we could have a libcockpit-noglib.a or something to avoid multiple compilation of these files (with different CFLAGS, no less)